### DISCUSSION TITLE
Error: Unable to find lambda for route

### DISCUSSION THREAD

**Discussion by Aymenjdily** [vercel/community#5121] [Help]
### Summary

anyone have an idea about this issue ? 

my Repo : https://github.com/Aymenjdily/Bags

### Steps to Reproduce

C:\Users\aymen\Desktop\Next.Js\ecommerce-bags>vercel build
Vercel CLI 32.6.1
WARNING: You should not upload the `.next` directory.
Installing dependencies...

up to date in 992ms

149 packages are looking for funding
  run `npm fund` for details
Detected Next.js version: 14.0.0
Detected `package-lock.json` generated by npm 7+
Running "npm run build"

> ecommerce-bags@0.1.0 build
> next build

   ▲ Next.js 14.0.0
   - Environments: .env

🌼   daisyUI 4.4.2
├─ ✔︎ 2 themes added            https://daisyui.com/docs/themes
╰─ ★ Star daisyUI on GitHub     https://github.com/saadeghi/daisyui

(node:20164) [DEP_WEBPACK_MODULE_UPDATE_HASH] DeprecationWarning: Module.updateHash: Use new ChunkGraph API
(Use `node --trace-deprecation ...` to show where the warning was created)
 ✓ Creating an optimized production build
 ✓ Compiled successfully
 ✓ Linting and checking validity of types
 ✓ Collecting page data
 ✓ Generating static pages (17/17)
 ✓ Collecting build traces
 ✓ Finalizing page optimization

Route (app)                              Size     First Load JS
┌ λ /                                    353 B           220 kB
├ ○ /_not-found                          885 B          88.7 kB
├ ○ /About                               352 B           220 kB
├ ○ /admin/login                         1.41 kB         140 kB
├ λ /api/order                           0 B                0 B
├ λ /api/order/[id]                      0 B                0 B
├ λ /api/product                         0 B                0 B
├ λ /api/product/[id]                    0 B                0 B
├ λ /api/user                            0 B                0 B
├ λ /api/user/[token]                    0 B                0 B
├ λ /api/user/login                      0 B                0 B
├ ○ /api/user/logout                     0 B                0 B
├ ○ /Checkout                            352 B           220 kB
├ ○ /dashboard                           107 kB          264 kB
├ λ /Details/[id]                        2.52 kB         172 kB
├ λ /orders                              3.39 kB         190 kB
├ λ /orders/[id]                         291 B           157 kB
├ λ /products                            3.12 kB         189 kB
├ λ /products/edit/[id]                  2.09 kB         198 kB
└ ○ /products/new                        2.09 kB         198 kB
+ First Load JS shared by all            87.8 kB
  ├ chunks/472-bc155e7e00d2839f.js       32.5 kB
  ├ chunks/fd9d1056-19be575faa0f315f.js  53.2 kB
  ├ chunks/main-app-d4f90f72148aa601.js  236 B
  └ chunks/webpack-7ea3dcaf1e1cfd60.js   1.93 kB

ƒ Middleware                             28.3 kB

○  (Static)   prerendered as static HTML
λ  (Dynamic)  server-rendered on demand using Node.js

Traced Next.js server files in: 455.894ms
Created all serverless functions in: 4.833s
Error: Unable to find lambda for route: /api/user/logout

**Comment by Aymenjdily** [comment]
@amyegan

**Comment by Aymenjdily** [comment]
@swarnava

**Comment by amyegan** [comment]
Hi @Aymenjdily. It looks like this is a duplicate of [your other post](https://github.com/orgs/vercel/discussions/5109). Please keep the discussion all in one place to make it easier to follow 🙏
